kapajira/test/test_kapacitor_utils.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+from kapajira.kapacitor.utils import AlertDataParser
+
+
+class TestAlertParser(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        with open('resources/alert_data.json') as f:
+            cls.ALERT_JSON_DATA = f.read()
+
+    def setUp(self):
+        self._alert_data = AlertDataParser.parse(TestAlertParser.ALERT_JSON_DATA)
+
+    def test_alert_data_navigation(self):
+        self.assertIn('HTTP_RESPONSE_TIME_HARD_SLA', self._alert_data.id)
+        self.assertIn('alerting-tester', self._alert_data.message)
+        self.assertEqual('CRITICAL', self._alert_data.level)
+        self.assertEqual('2016-11-28T10:24:00Z', self._alert_data.time)
```

Comment:
I think the most complex code in this PR is in AlertData. __getattr__/build methods - it can return lists or mappings. Yet, I don't see a getter test for attributes hidden deeper in the JSON config.